app/views/compute_profiles/index.html.erb

```diff
@@ -14,7 +14,6 @@
       <tr>
         <td class="ellipsis"><%= link_to_if_authorized compute_profile.name, hash_for_compute_profile_path(compute_profile).merge(:auth_object => compute_profile, :authorizer => authorizer, :auth_action => 'edit') %></td>
         <td><%= action_buttons(
-                  display_link_if_authorized(_('Edit'), hash_for_compute_profile_path(compute_profile).merge(:auth_object => compute_profile, :authorizer => authorizer, :auth_action => 'edit')),
```

Comment:
The profile name already links to the edit page. On other pages, we show an "Edit" button only if clicking on the element name leads to a separate "show" page.